Projects/Offer/Sources/Components/MainContentForm/Sections/CurrentInsurerSection.swift

```diff
@@ -0,0 +1,113 @@
+import Flow
+import Form
+import Foundation
+import Presentation
+import UIKit
+import hCore
+import hCoreUI
+import hGraphQL
+
+struct CurrentInsurerSection {
+	let quoteBundle: GraphQL.QuoteBundleQuery.Data.QuoteBundle
+}
+
+extension CurrentInsurerSection: Presentable {
+	func materialize() -> (UIView, Disposable) {
+		let bag = DisposeBag()
+
+		let sectionContainer = UIStackView()
+		sectionContainer.axis = .vertical
+
+		bag += sectionContainer.addArranged(Spacing(height: 16))
+
+		bag += {
+			for view in sectionContainer.subviews {
```

Comment:
whats the point of this?

Reply:
Yeah thought it was needed for proper disposal of the views, but not needed

Reply:
Apparently it's needed 😅